website/data/cdktf-nav-data.json

```diff
@@ -173,27 +173,132 @@
     "routes": [
       {
         "title": "Overview",
-        "path": "api-reference/overview"
+        "path": "api-reference"
       },
       {
         "title": "Typescript",
-        "path": "api-reference/typescript"
```

Comment:
Same as for the overview page – can / should we add redirects to just `api-reference`?